src/main/java/org/traccar/notificators/NotificatorTelegram.java

```diff
@@ -70,9 +108,4 @@ public void failed(Throwable throwable) {
         });
     }
 
-    @Override
-    public void sendAsync(long userId, Event event, Position position) {
```

Comment:
What happened to this method?

Reply:
This method is safe and sound, you may [check the class file in my repo](https://github.com/soshial/traccar/blob/patch-1/src/main/java/org/traccar/notificators/NotificatorTelegram.java).
Github diff algorithms are not ideal.

Reply:
Can we move it around, so GitHub doesn't show it as a change?